File: triforce.py
from sys import stdin

numQueries=int(stdin.readline())
for i in range(1,numQueries+1):
    #Read input for testcase
    s,r,p=stdin.readline().split()
    s=float(s)
    r=int(r)
    p=int(p)

    #Compute the area of an equilateral triangle
    #Area of a triangle is base*height, and 
    #height = base*sqrt(3)/2
    area=s*s/4*(3**.5)
    #Each recursive step for the pattern has 3/4 of the original area.
    #This is evident if you look at the sample, going from 0 to 1 triforce.
    #So, we just multiply by 3/4 each time we recurse. The area will
    #continue to decrease
    area*=pow(3.0/4,r)   
    #Once we have our final area, multiply by price
    area*=p
    
    #Print the rounded area
    print("Triforce #%d: %.2f"%(i,area))
    numQueries-=1

File: triforce.java
import java.util.Scanner;

public class triforce {
    public static void main(String[] args) {
        double sqrt3 = Math.sqrt(3);
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        for(int c = 0; c < n; c++) {
            int s = in.nextInt();
            int t = in.nextInt();
            int p = in.nextInt();
            if(t > 100) {
                System.out.printf("Triforce #%d: 0.00\n", c+1);
                continue;
            }
            double area = sqrt3*s*s/4*Math.pow(3.0/4.0, t);
            System.out.printf("Triforce #%d: %.2f\n", c+1, area*p);
        }
    }
}
